graphics/fonts/macfont.cpp

```diff
@@ -503,11 +503,14 @@ MacFONTFont *MacFONTFont::scaleFont(const MacFONTFont *src, int newSize, int sla
 		return NULL;
 	}
 
-	Graphics::Surface srcSurf, tmpSurf;
-	srcSurf.create(MAX(src->getFontSize() * 2, newSize * 2), MAX(src->getFontSize() * 2, newSize * 2),
-				PixelFormat::createFormatCLUT8());
-	int dstGraySize = newSize * 20 * newSize;
-	int *dstGray = (int *)malloc(dstGraySize * sizeof(int));
+	BdfFont* srcCopy = new BdfFont(src->_data, DisposeAfterUse::YES);
+	srcCopy->initializeScaling(src->getFontSize(), newSize);
+
+	Graphics::Surface tmpSurf;
+	// srcSurf.create(MAX(src->getFontSize() * 2, newSize * 2), MAX(src->getFontSize() * 2, newSize * 2),
```

Comment:
Why is this still commented out instead of being removed?

Reply:
fixed that